Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenWindowControllerIOS.mm

```diff
@@ -871,6 +871,9 @@ - (void)enterFullScreen:(CGSize)videoDimensions
     page->setSuppressVisibilityUpdates(true);
 
     _viewState.store(webView.get());
+#if PLATFORM(VISION)
+    [webView _setViewScale:1.f];
```

Comment:
Do we really want to set the scale before taking the snapshot?

Reply:
This change isn't needed. It is fine to set the scale below where we set mEDW. I'll remove this change